dashboard/src/locales/messages/index.ts

```diff
@@ -76,11 +76,15 @@ export const messages = {
     'filter.configsSubtitle': 'Please select one or more configs:',
     'filter.durationSubtitle': 'Please select the duration range:',
     'filter.filtering': 'Filtering',
+    'filter.globalFilter': 'Global filters',
+    'filter.globalFilterAllTabs': 'Global filters affects all the tabs',
     'filter.hardware': 'Hardware',
-    'filter.hardwareSubtitle': 'Please select one or more hardwares:',
+    'filter.hardwareSubtitle': 'Please select one an specific hardwares:',
```

Comment:
this phrase doesn't make a lot of sense and I think you should keep the "or more", maybe `Please select one or more specific hardwares`

Reply:
oh this was probably a missplaced copy and paste